### ISSUE TITLE
update software.sil.org with latest download available

### ISSUE THREAD

**Issue by megahirt** [sillsdev/SpeechAnalyzer#25]
Update https://software.sil.org/speech-analyzer/#downloads

**Comment by darcywong00** [comment]
@megahirt will handle this for the latest [v3.1.1.1 release](https://github.com/sillsdev/SpeechAnalyzer/releases/tag/v3.1.1.1).

**Comment by megahirt** [comment]
software.sil.org/speech-analyzer is now updated with the latest download version.  I had to get SFTP access in order to upload the release to the /downloads folder and modify the SpeechAnalyzer.download_info file which controls the green download button on the right hand side.  Unfortunately, the green download widget does not work with github-hosted released files.  It is only capable of linking to relative paths which means the file needs to be uploaded to software.sil.org first.

![image](https://user-images.githubusercontent.com/3444521/130306866-db4083c0-b6df-4338-8ed6-c574b95a8dc9.png)


![image](https://user-images.githubusercontent.com/3444521/130306875-13032add-bec8-4d99-b52c-3e484c47deaf.png)
